frontend/packages/citizen-frontend/src/map/MapView.tsx

```diff
@@ -214,15 +214,7 @@ const FlexContainer = styled(AdaptiveFlex)`
   margin-bottom: ${defaultMargins.m};
   align-items: stretch;
 
-  width: 1520px;
-
-  @media (max-width: 1900px) {
-    width: 80%;
-  }
-
-  @media (max-width: 1700px) {
-    width: 90%;
-  }
+  width: 90%;
```

Comment:
If we want to remove all the paddings as well I tried something like this:

```
const FullScreen = styled.div`
  position: absolute;
  top: 0;
  bottom: 0;
  left: 0;
  right: 0;
  display: flex;
  justify-content: center;
  align-items: stretch;
`

const FlexContainer = styled(AdaptiveFlex)`
  margin-top: 64px;
  align-items: stretch;

  width: 100%;

  @media (max-width: ${mapViewBreakpoint}) {
    margin-top: ${headerHeight};
    width: 100%;
    margin-bottom: 0;
    &.mobile-mode-map {
      .unit-list {
        display: none;
      }
    }
    &.mobile-mode-list {
      .map-box {
        display: none;
      }
    }
  }
`
```

The main changes: 

- `width: 100%` for FlexContainer.
- `top 0` for `FullScreen`.
- `margin-top: 64px` for FlexContainer and `margin-top: ${headerHeight}` for mobile sized FlexContainer to take into account different header sizes on mobile and desktop

Reply:
Looks good. I'll still check from UX whether to use width: 95% or 100%